lib/config.js

```diff
@@ -148,9 +148,11 @@ function readConfigFromFile(filePath) {
             filePath = "eslint-config-" + filePath;
         }
 
-        // TODO: support other configs
-        // config = util.mergeConfigs(config, require(filePath));
+        if (filePath.match(/^eslint-config-airbnb.*/)) {
```

Comment:
Can you check that this configuration doesn't require any plugins we don't currently support?